<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="upload_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>upload one file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compress_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>start compressing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uncompress_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>decompress the file</string>
    </property>
   </widget>
   <widget class="QLabel" name="upload_file_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>uploaded file name：</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_name_lable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>571</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>choose one file</string>
    </property>
   </widget>
   <widget class="QLabel" name="compress_file_lable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>compressed file path：</string>
    </property>
   </widget>
   <widget class="QLabel" name="compressed_file_lable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="choose_file_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>file chosen：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manager_login_btn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>manager login</string>
    </property>
   </widget>
   <widget class="QLabel" name="compress_file_lable_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>decompressed file path：</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_dir_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>choose one folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uncompress_batch_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>decompress the folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose_dir_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>folder chosen：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
